src/main/webapp/script.js

```diff
@@ -0,0 +1,114 @@
+
+import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
+
+// global objects
```

Comment:
Comment should starts with capital and ends with period.
The same as other places.